deploy/kubernetes/overlays/prow-gke-release-staging-rc/kustomization.yaml

```diff
@@ -8,10 +8,10 @@ images:
   newTag: "v0.6.0-rc1"
```

Comment:
do we need a new pd driver rc too? @davidz627

Reply:
Updated